functions.yml

```diff
@@ -350,7 +367,8 @@ quality_control:
           [Row(stats=[Row(sampleId='NA12878', mean=2.0, stdDev=1.0, min=1.0, max=3.0)])]
       args:
         - name: genotypes
-          doc: The array of genotype structs
+          doc: An array of genotype structs
```

Comment:
`An array of genotype structs` -> `An array of genotype structs with a ``depth`` field`

Reply:
Done